package.json

```diff
@@ -3,46 +3,28 @@
   "version": "1.0.0",
   "description": "SLA Application using node-ari-client.js",
```

Comment:
As far as NPM is concerned, the client is ari-client. node-ari-client is only the name on github.